TMDS_3/include/CGAL/tetrahedron_soup_to_triangulation_3.h

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) 2009-2014 INRIA Sophia-Antipolis (France).
+// Copyright (c) 2010-2013 GeometryFactory Sarl (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+//
+// Author(s)     :  Mael Rouxel-Labbé, Maxime Gimeno, Jane Tournois
+//
+//******************************************************************************
+// File Description :
+//******************************************************************************
+
+#ifndef CGAL_TMDS_3_TETRAHEDRON_SOUP_TO_C3T3_H
+#define CGAL_TMDS_3_TETRAHEDRON_SOUP_TO_C3T3_H
+
+#include <CGAL/license/TMDS_3.h>
+
+#include <CGAL/TMDS_3/tet_soup_to_c3t3.h>
+#include <CGAL/TMDS_3/internal/TMDS_3_helper.h>
+#include <CGAL/Named_function_parameters.h>
+#include <CGAL/boost/graph/named_params_helper.h>
+
+#include <vector>
+#include <array>
+#include <boost/unordered_map.hpp>
+
+namespace CGAL {
+
+  /** \ingroup PkgTMDS3Functions
+  * builds a 3D triangulation from a soup of tetrahedra.
+  *
+  * @tparam TetrahedronRange a model of `Range` whose value type is
+  * a `Tetrahedron_3`. The point type of the tetrahedra must be convertible
+  * to `Triangulation::Point`
+  * @tparam Triangulation a 3D triangulation class that has
+  * a vertex base model of `MeshVertexBase_3`
+  * and a cell base model of `MeshCellBase_3`
+  *
+  * @param tets the set of finite tetrahedra of a valid CGAL triangulation.
+  * Each element in the range is the geometric description of the
+  * corresponding cell in `tr`
```

Comment:
```suggestion
  * corresponding cell in `tr`.
```

Reply:
`tr` does not exist anymore since it returns the triangulation (same in the other function).